packages/driver/src/dom/coordinates.ts

```diff
@@ -12,7 +12,19 @@ const isAutIframe = (win) => {
 
   // https://github.com/cypress-io/cypress/issues/6412
   // ensure the parent is a Window before checking prop
-  return $window.isWindow(parent) && !$elements.getNativeProp(parent, 'frameElement')
+  if (!$window.isWindow(parent)) {
```

Comment:
Is there anyway to add a test or test this in `coordinates_spec.ts`? I feel like this might be difficult currently.